crates/rsharp/rust/src/primitives.rs

```diff
@@ -14,7 +14,7 @@ macro_rules! primitives {
             bool,
             f32,
             f64,
-            // String,
+            reprc_string,
```

Comment:
you should just say `String`, this macro is usually used both in places that expect c_types::T and just a regular rust T.

Reply:
I created this type alias since the `ReprC` trait isn't defined for `std::String` - see below. What would you recommend to fix this?

<img width="892" alt="image" src="https://user-images.githubusercontent.com/5103968/184447939-3f1b5adc-db4d-4da6-8d50-4ee5954d6994.png">

Reply:
https://github.com/0xTELEPORTAL/tp-platform/blob/deafa867e4b25f4533a9148954fd4c365bb38c8e/client/rust/src/contract/properties/mod.rs#L291

Currently this is in the tp_client crate, but it could/should have been moved to `rsharp`. Just leave it there however, and reference the type in that link.

Reply:
Wouldn't that create a cyclic package dependency?

Reply:
far point, rsharp should not have tp_client as a dependency. You can move the String definition from tp_client into rsharp.

Reply:
Created SER-459 to track the "move string from tp_client into rsharp" task